### DISCUSSION TITLE
Add possibility to scan for copying pre-built .xctestrun file into test_output directory

### DISCUSSION THREAD

**Discussion by Kukurijek** [fastlane/fastlane#18056] [Ideas] [answered]
# Feature Request
It would be nice if scan supported the option to copy .xctestrun file into test_output directory.

**Motivation Behind Feature**
Running UI tests on a third-party service (e.g. Firebase Testlab) requires a build_products.zip file, which contains a .xctestrun configuration file. 

If you want to dynamically insert some settings (e.g. environment variable) when uploading the zip file , you have to unzip the file, change the .xctestrun file (which usually always has a different name) and compress everything again. 
Most likely you will do it in a container, which means that you have to install additional libraries unnecessarily.
If you use [gcloud](https://cloud.google.com/sdk/gcloud/reference/firebase/test/ios/run), which is quite convenient, you could just pass new .xctestrun file as parameter, so you do not need to zip or unzip anything.

**Feature Description**
If copy of .xctestrun file (with some default name) is provided in test_output directory, process of delivering UI tests would be much easier.

**Comment by rogerluan** [comment, marked as answer]
This was implemented here https://github.com/fastlane/fastlane/pull/18057
